### PR TITLE
Last of error codes documentation

### PR DESCRIPTION
**Pull request by drwill-ms**

### CODE REVIEW

File: iothub/service/src/Common/Exceptions/ErrorCode.cs [hunk 1]
```diff
@@ -7,12 +7,10 @@
 namespace Microsoft.Azure.Devices.Common.Exceptions
 {
     /// <summary>
-    /// Error codes for common IoT hub exceptions.
+    /// Error codes for common IoT hub response errors.
```
Comment (drwill-ms) [inline iothub/service/src/Common/Exceptions/ErrorCode.cs:10]: Technically speaking, we get these errors in responses; they aren't exceptions.

File: iothub/service/src/Common/Exceptions/ErrorCode.cs [hunk 2]
```diff
@@ -7,12 +7,10 @@
...
     /// </summary>
     public enum ErrorCode
     {
-#pragma warning disable CS1591 // Missing XML comment for publicly visible type or member
```
Comment (drwill-ms) [inline iothub/service/src/Common/Exceptions/ErrorCode.cs:14]: Last of undocumented members!

File: iothub/service/src/Common/Exceptions/ErrorCode.cs [hunk 3]
```diff
@@ -93,7 +91,7 @@ public enum ErrorCode
         /// <summary>
         /// A devices with the same Id was present multiple times in the input request for bulk device registry operations.
         /// <para>
-        /// For more information <see href="https://docs.microsoft.com/rest/api/iothub/service/bulk-registry/update-registry"/> on bulk registry operations.
+        /// For more information on bulk registry operations, see <see href="https://docs.microsoft.com/rest/api/iothub/service/bulk-registry/update-registry"/>.
```
Comment (drwill-ms) [inline iothub/service/src/Common/Exceptions/ErrorCode.cs:94]: Making consistent verbiage. See others.

File: iothub/service/src/Common/Exceptions/ErrorCode.cs [hunk 4]
```diff
@@ -233,32 +239,43 @@ public enum ErrorCode
         DeviceMessageLockLost = 412002,
 
         // RequestEntityTooLarge - 413
```
Comment (drwill-ms) [inline iothub/service/src/Common/Exceptions/ErrorCode.cs:241]: Copying elsewhere where we have a newline after a section header and enum values.

File: iothub/service/src/Common/Exceptions/ErrorCode.cs [hunk 5]
```diff
@@ -233,32 +239,43 @@ public enum ErrorCode
...
+
         /// <summary>
         /// When the message is too large for IoT hub you will receive this error. You should attempt to reduce your message size and send again.
-        /// For more information on message sizes, see <see href="https://docs.microsoft.com/azure/iot-hub/iot-hub-devguide-quotas-throttling#other-limits">IoT hub quotas and throttling | Other limits</see>
+        /// For more information on message sizes, see <see href="https://aka.ms/iothubthrottling#other-limits">IoT hub quotas and throttling | Other limits</see>
```
Comment (drwill-ms) [inline iothub/service/src/Common/Exceptions/ErrorCode.cs:245]: Using the aka consistently to save characters.

Comment (drwill-ms) [reply iothub/service/src/Common/Exceptions/ErrorCode.cs:245]: I checked to make sure section references (#other-limits) works when using the aka.

**Review by jamdavi — APPROVED** [review]

**Pull request merged**